composer.json

```diff
@@ -149,7 +148,6 @@
         "symfony/security-csrf": "^6.4",
         "symfony/security-http": "^6.4",
         "symfony/service-contracts": "^3.0",
-        "symfony/stopwatch": "^6.4",
```

Comment:
Are we sure we can remove this? We explicitly readded it in https://github.com/contao/contao/pull/1093
I’m not sure why though. Maybe this is used by the web profiler debug toolbar?

Reply:
You are right, the time panel will be empty without it.

<img width="275" alt="" src="https://github.com/contao/contao/assets/1192057/839a33eb-26a8-423e-b4aa-19fbe0019476">

Added again in 7619568aadb86bbec9f8361f07346a697c2aafab.

Reply:
We also use the stopwatch directly in our code - but as a soft dependency, and thus is only required by the `contao/manager-bundle` and not the `contao/core-bundle`.

Reply:
We could add a comment on "why are we requiring this dependency" for every `require` in the `extra` section and enforce this using some tool. I have seen other developers suggesting this as well.